<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatFormatWidget</class>
 <widget class="QWidget" name="ChatFormatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>对话整理工具 - 支持自定义分隔符</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>文件设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="filePathEdit">
        <property name="placeholderText">
         <string>请选择对话文件...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>名称设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>您的名字：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="myNameEdit">
        <property name="placeholderText">
         <string>请输入您的名字</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>AI的名字：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="aiNameEdit">
        <property name="placeholderText">
         <string>请输入AI的名字</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>分隔符设置（支持转义字符：\n 换行, \t 制表符, \r 回车）</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="toolTip">
         <string>用于分隔人类对话和AI对话的分隔符（默认：edit）</string>
        </property>
        <property name="text">
         <string>AI/人类分隔符：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>用于分隔不同对话块的分隔符（默认：两个换行 \\n\\n）</string>
        </property>
        <property name="text">
         <string>段间分隔符：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="blockSeparatorEdit">
        <property name="toolTip">
         <string>用于分隔不同对话块的分隔符（默认：两个换行 \\n\\n）</string>
        </property>
        <property name="placeholderText">
         <string>例如：\n\n（两个换行）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="aiHumanSeparatorEdit">
        <property name="toolTip">
         <string>用于分隔人类对话和AI对话的分隔符（默认：edit）</string>
        </property>
        <property name="placeholderText">
         <string>例如：edit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="resetSeparatorsButton">
        <property name="text">
         <string>恢复默认分隔符</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="formatButton">
        <property name="text">
         <string>开始整理</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>处理日志</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="logTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
